src/github/entities/user-installation.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class UserInstallation extends Base {
+  @Column({ type: 'varchar', length: 20 })
+  installationId: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  organizationName: string;
+
+  @ManyToOne(() => User, (user) => user.installations)
+  user: User;
+}
```

Comment:
PullProofAI

Consider adding validation decorators to ensure data integrity and consistency.
```diff
+   @IsNotEmpty()
+   @IsString()
+   @Column({ type: 'varchar', length: 20 })
+   installationId: string;

+   @IsNotEmpty()
+   @IsString()
+   @Column({ type: 'varchar', length: 255 })
+   organizationName: string;

+   @ManyToOne(() => User, (user) => user.installations)
+   user: User;
```
This addition ensures that the `installationId`, `organizationName`, and `user` properties are not empty and are of type string.



<!-- This is an auto-generated comment by PullProofAI -->